compiler/testData/codegen/bytecodeText/nullCheckOptimization/localLateinit/initialized.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
This test is now failing because it contain an IFNULL instruction. However, it only passed before because the IFNULL was an IFEQ comparing to the null constant.